backend/src/tests.rs

```diff
@@ -170,11 +139,20 @@ mod test {
             ("0xb17a9e25265d3b88de7bfad81e7accad6e3d5612308ff83cc0fef76a34152b0444309e8fc3dea5139e49b6fc83a8553071a7af3d0cfd3fb8c1aea2a4c171729c1c").parse().unwrap(),
             message:  "Summa proof of solvency for CryptoExchange".encode().into(),
         }];
-        let result = summa_signer
-            .submit_proof_of_address_ownership(owned_addresses)
+
+        let mut address_ownership_client = AddressOwnership::new(
```

Comment:
I think that the address ownership should be a part of the Round (similarly to Snapshot). While it's possible to submit it at any time before the proof, it's still enough to submit it in the beginning of a round. Most likely it wouldn't be called manually in the production implementation of Summa like you're calling it here. There should be some entity that controls it and Round seems to be a good candidate for that.

Reply:
https://github.com/summa-dev/summa-solvency/pull/152#discussion_r1321759814 partially addresses your suggestion for now.
A potential alternative could be the `Scheduler` you introduced in the [Service-Based Backend Architecture](https://hackmd.io/CXoZfrc0Ri-UOd-whxvvnA)
Let's discuss this further in the document.